crates/chain-listener/src/listener.rs

```diff
@@ -794,46 +795,31 @@ impl ChainListener {
         Ok(())
     }
 
-    /// Return already started units involved in CC and not having less than MIN_PROOFS_PER_EPOCH proofs in the current epoch
-    fn get_priority_units(&self) -> Vec<CUID> {
-        self.cc_compute_units
-            .iter()
-            .filter(|(_, cu)| cu.startEpoch <= self.current_epoch) // CU is already started
-            .filter(|(cuid, _)| {
-                self.proof_counter
-                    .get(cuid)
-                    .map(|count| *count < self.min_proofs_per_epoch)
-                    .unwrap_or(true)
-            })
-            .map(|(cuid, _)| *cuid)
-            .collect()
-    }
-
-    /// Return already started units involved in CC and found at least MIN_PROOFS_PER_EPOCH proofs,
-    /// but less that MAX_PROOFS_PER_EPOCH proofs in the current epoch
-    fn get_non_priority_units(&self) -> Vec<CUID> {
-        self.cc_compute_units
-            .iter()
-            .filter(|(_, cu)| cu.startEpoch <= self.current_epoch) // CU is already started
-            .filter(|(cuid, _)| {
-                self.proof_counter
-                    .get(cuid)
-                    .map(|count| {
-                        *count >= self.min_proofs_per_epoch && *count < self.max_proofs_per_epoch
-                    })
-                    .unwrap_or(true)
-            })
-            .map(|(cuid, _)| *cuid)
-            .collect()
-    }
-
-    /// Return units in CC that is not active yet and can't produce proofs in the current epoch
-    fn get_pending_units(&self) -> Vec<CUID> {
-        self.cc_compute_units
-            .values()
-            .filter(|cu| cu.startEpoch > self.current_epoch) // CU hasn't yet started
-            .map(|cu| CUID::new(cu.id.0))
-            .collect()
+    fn group_cc_units(&self) -> CUGroupResult {
```

Comment:
```suggestion
    fn get_cu_groups(&self) -> CUGroupResult {
```

or `get_cu_layout`

`group_cc_units` has the following problems: verb is `group`, implying function is `&mut`; `cc_units` term doesn't exist in our lingo.

Reply:
Fixed